{"name": "Проверка создания пользователя, который уже зарегистрирован.", "status": "passed", "start": 1728853859021, "stop": 1728853859393, "uuid": "1981ae54-9023-4472-bb90-e359c84b17d7", "historyId": "511af5c3d1ba0594e424965a597653ae", "testCaseId": "511af5c3d1ba0594e424965a597653ae", "fullName": "tests.test_create_user.TestCreateUser#test_create_double_user_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "Vladimir"}, {"name": "thread", "value": "515800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}